### ISSUE TITLE
Automatic Concluded License Decider Issue

### ISSUE THREAD

**Issue by DineshDevarajan** [fossology/fossology#1584]
### Description

While uploading a file with Automatic Concluded License Decider with scanners matches if Ojo findings are no contradiction with other findings. The license getting auto concluded is made global scope and due to this same file when uploaded twice with and without Automatic Concluded License Decider both of them are showing concluded licenses.

#### How to reproduce

1. Upload a project without decider ( say uploadid = 1 )
2. Upload same project with Automatic Concluded License Decider(OJO) and check the license is getting auto concluded ( say uploadid = 2 )
3.  Now got to the upload ( uploadid = 1 ) and check the file here also it is auto concluded due to global scope

### Versions

3.7.0

 Is this an expected behavior ?

**Comment by GMishx** [comment]
@DineshDevarajan, thank you for the ticket. This was not an expected behavior.

It will be fixed by #1596

**Comment by GMishx** [comment]
I am sorry, may be I commented prematurely.
The agent `nomos` is most trusted agent. And with `ojo`, the findings are done using `SPDX-License-Identifier` tag which has clear indication about the licensing of the file. Also, both agents works on different algorithms.
Therefore, it was decided to keep the decisions as global as it is extremely rare that both the agents will be wrong for a given file.


@shaheemazmalmmd, would you like to add some comments?

**Comment by shaheemazmalmmd** [comment]
@GMishx Agreed. I just recalled the discussion we have had with @mcjaeger, while implementing this feature.
